lib/reline/windows.rb

```diff
@@ -85,7 +85,7 @@ def initialize(dllname, func, import, export = "0", calltype = :stdcall)
       def call(*args)
         import = @proto.split("")
         args.each_with_index do |x, i|
-          args[i], = [x == 0 ? nil : x].pack("p").unpack(POINTER_TYPE) if import[i] == "S"
+          args[i], = [x == 0 ? nil : +x].pack("p").unpack(POINTER_TYPE) if import[i] == "S"
```

Comment:
This is for ruby 2.6 compatibility, because it has a bug in `pack("p")`:

```
$ ruby -v -e '["a".freeze].pack("p")'
ruby 2.6.10p210 (2022-04-12 revision 67958) [universal.arm64e-darwin23]
-e:1:in `pack': can't modify frozen String (FrozenError)
	from -e:1:in `<main>'
```

Seem to have been fixed in 2.7